### PR TITLE
PBL-169: Add Alert when User doesn't have enough KSM on USDT Picasso=>Statemine

### PR DESCRIPTION
**Pull request by easteregg**

### CODE REVIEW

File: frontend/apps/picasso/pages/transfers/index.tsx [hunk 1]
```diff
@@ -43,7 +46,22 @@ const Transfers: NextPage = () => {
...
+      to === "statemine" &&
+      ksmBalance.lt(
+        fromChainIdUnit(
+          PICASSO_STATEMINE_KSM_TRANSFER_FEE,
+          tokens.ksm.decimals.picasso
+        )
+      ) &&
+      selectedToken === "usdt",
+    [from, to, ksmBalance, tokens.ksm.decimals.picasso, selectedToken]
+  );
```
Comment (fl-y) [inline frontend/apps/picasso/pages/transfers/index.tsx:64]: Does useMemo make sense here? On a quick skim it doesn't seem so to me

reference: https://blog.logrocket.com/rethinking-hooks-memoization/

File: frontend/apps/picasso/pages/transfers/index.tsx [hunk 2]
```diff
@@ -146,6 +164,11 @@ const Transfers: NextPage = () => {
         <Grid item {...gridItemStyle("1.5rem")}>
           <TransferRecipientDropdown />
         </Grid>
+        {isKSMAlertVisible ? (
+          <Grid item {...gridItemStyle("1.5rem")}>
+            <TransferKSMAlert />
+          </Grid>
+        ) : null}
```
Comment (fl-y) [inline frontend/apps/picasso/pages/transfers/index.tsx:171]:
```suggestion
        {isKSMAlertVisible && (
          <Grid item {...gridItemStyle("1.5rem")}>
            <TransferKSMAlert />
          </Grid> 
        )}
```
unecessary ternary

**Review by fl-y — APPROVED** [review]
Left comments that do not necessarily need to be addressed

File: frontend/apps/picasso/pages/transfers/index.tsx (hunk 2 above)
Comment (easteregg) [reply frontend/apps/picasso/pages/transfers/index.tsx:171]: Conditional rendering with logical operators has a tendency to create issues later on if we don't be careful with it to make sure it is always boolean. There is always the danger of short-circuiting when the parent condition changes. 

That's why I have chosen the safest, but, more verbose route.

File: frontend/apps/picasso/pages/transfers/index.tsx (hunk 1 above)
Comment (easteregg) [reply frontend/apps/picasso/pages/transfers/index.tsx:64]: You are right, I usually don't overuse memoization, but had a gut feeling this one needed it. Will remove, retest and post update here.

Comment (easteregg) [reply frontend/apps/picasso/pages/transfers/index.tsx:64]: You got it 100%. it doesn't add any value. removing it.

**Comment by easteregg** [comment]
Closing in favor of https://github.com/ComposableFi/composable/pull/2963

**Pull request closed without merging**